backends/tfhe-cuda-backend/cuda/src/integer/integer.cuh

```diff
@@ -2364,11 +2370,15 @@ __host__ void integer_radix_apply_noise_squashing_kb(
         ksks, lut->input_big_lwe_dimension, small_lwe_dimension, ks_base_log,
         ks_level, lwe_array_out->num_radix_blocks);
 
-    execute_pbs_128_async<__uint128_t>(
-        streams, gpu_indexes, active_gpu_count, lwe_after_pbs_vec, lut->lut_vec,
-        lwe_after_ks_vec, bsks, ms_noise_reduction_key, lut->pbs_buffer,
-        small_lwe_dimension, glwe_dimension, polynomial_size, pbs_base_log,
-        pbs_level, lwe_array_out->num_radix_blocks);
+    /// int_noise_squashing_lut doesn't support a different output or lut
+    /// indexing than the trivial
+    execute_pbs_async<uint64_t, __uint128_t>(
```

Comment:
I'm not sure we need to have the two template types: the PBS128 always takes u64 as input and returns u128. This complicates a bit the code so I'm not sure we need it.

Reply:
I moved this entire refactor to https://github.com/zama-ai/tfhe-rs/pull/2692 where we can better discuss this change. 128-bit decompression will not execute a PBS anyway, so there is no need to have this here. This was actually an artifact from when I started working on this and didn't know about this difference.